build-system/tasks/e2e/network-logger.js

```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import selenium from 'selenium-webdriver';
+const selenium = require('selenium-webdriver');
```

Comment:
When loading this file in a worker this isn't being seen as a module. That might be solvable but I'm not sure it's worth it